src/ecfeed.hpp

```diff
@@ -0,0 +1,1064 @@
+#pragma once
```

Comment:
please replace it with classic include guards (#ifndef ECFEED_HPP...)

Reply:
Done.